x-pack/plugins/fleet/server/routes/epm/handlers.ts

```diff
@@ -650,6 +652,33 @@ export const reauthorizeTransformsHandler: FleetRequestHandler<
   }
 };
 
+export const getInputsHandler: FleetRequestHandler<
+  TypeOf<typeof GetInputsRequestSchema.params>,
+  TypeOf<typeof GetInputsRequestSchema.query>,
+  undefined
+> = async (context, request, response) => {
+  const soClient = (await context.fleet).internalSoClient;
+
+  try {
+    const { pkgName, pkgVersion } = request.params;
+    const { format } = request.query;
+    if (format === 'json') {
+      const res = await getTemplateInputs(soClient, pkgName, pkgVersion, 'json');
+      return response.ok({ body: res });
+    } else if (format === 'yml' || format === 'yaml') {
```

Comment:
Good call to permit both `yml` and `yaml` - we are inconsistent across the product with which we prefer.